lib/loadable_config.rb

```diff
@@ -28,7 +28,7 @@ def self.attribute(attr, type: :string, optional: false)
     attr_accessor attr
 
     singleton_class.instance_eval do
-      delegate attr, to: :instance
+      define_method(attr) { self.class.send(attr) }
```

Comment:
Does this work? The `delegate` call is to create a class method which forwards to the `Singleton` instance.